browser/index.ts

```diff
@@ -56,6 +56,7 @@ const getNewPdfName = () => {
 export const previewPdf = async (
   url: string,
   template: ServiceNames,
+  orientationOption: boolean,
```

Comment:
This one should be optional `orientationOption?: boolean` or have a default value.